### PR TITLE
Mention file-format-changelog.rst in CONTRIBUTING

### PR DESCRIPTION
**Pull request by fgaz**
As proposed in #hackage

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -226,6 +226,9 @@ You can find a large number of real-world examples of changelog files
 At release time, the entries will be merged with
 [this tool](https://github.com/phadej/changelog-d).
 
+In addition, if you're changing the .cabal file format specification you should
```
Comment (Mikolaj) [inline CONTRIBUTING.md:229]: How about cabal.project file format? That's not a part of Cabal the library?

Comment (fgaz) [reply CONTRIBUTING.md:229]: no, that's only a cabal-install thing (though it's based on Cabal's field grammar)

**Review by Mikolaj — APPROVED** [review]

**Review by jneira — APPROVED** [review]
thanks

**Comment by Mikolaj** [comment]
Tests got canceled. Retrying.

**Comment by Mikolaj** [comment]
@mergifyIO refresh

**Comment by Mikolaj** [comment]
@mergifyIO rebase

**Pull request merged**